testutil/testutil.go

```diff
@@ -2,21 +2,17 @@ package testutil
 
 import (
 	"context"
+	"os"
 
 	batchv1 "k8s.io/api/batch/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func InitPresetModelImageConfigMap() *corev1.ConfigMap {
-	presetImageConfigMap := &corev1.ConfigMap{}
-	presetImageConfigMap.Data = map[string]string{
-		"savedmodel-extract":       "demo.goharbor.com/release/savedmodel:v0.2",
-		"ormb-storage-initializer": "demo.goharbor.com/release/savedmodel:v0.2",
-	}
-
-	return presetImageConfigMap
+func InitPresetModelImage() {
+	os.Setenv("SAVEDMODEL_EXTRACT_IMAGE", "demo.goharbor.com/release/savedmodel:v0.2")
+	os.Setenv("ORMB_INITIALIZER_IMAGE", "demo.goharbor.com/release/savedmodel:v0.2")
```

Comment:
ping /cc @simon-cj I'm not sure about the image in here.

Reply:
Tt will check string format, not check the content.
Maybe there need all of the env key for all image.